README.md

```diff
@@ -36,21 +26,17 @@ import "github.com/Fantom-foundation/Carmen/go/carmen"
 
 
 # How to Use
-Carmen is configured for the scheme to use, and if historical data are retained or not. The Merkle-Patricia-Trie 
-compatible scheme is called S5, and it can be either Archive or non-Archive (LiveDB).
+Schema S5 is the only schema compatible with Merkle-Patricia Tries. Carmen has always enabled LiveDB to keep the state of the last block.
+ArchiveDB can be enabled to retain historical data. If ArchiveDB is enabled, the state of the last block is appended to the archive for later retrieval. 
 
-A state of the blockchain may be updated always only at the head of the chain, and it is stored in the LiveDB. 
-The state becomes verbatim as part of a block once appended to the blockchain via the new block. If the Archive mode 
-is enabled, the state of the last block is appended to the archive and it can be retrieved later. If the Archive 
-is not configured, only the head state is available. LiveDB is enabled all the time, while the Archive is tentative.
-
-As a blockchain is a sequence of blocks where each contains a set of transactions, the API is oriented 
-to creation and retrieval of transactions within blocks. Carmen provides an API for accessing both Live 
-and Archive databases to append new blocks to the blockchain and to query history for previous blocks.
+The Carmen interface is loosely related to the go-ethereum's StateDB interface, though it differs w.r.t. block management and transaction management. 
```

Comment:
not sure if this change improves this paragraph. First it IMHO unnecessarily refers to ethereum's StateDB and then suddenly it becomes too brief